git_wrapper/tag.py

```diff
@@ -72,3 +72,14 @@ def push(self, name, remote, dry_run=False):
                 self.git_repo.git.push(remote, name)
         except git.GitCommandError as ex:
             raise_from(exceptions.PushException(msg), ex)
+
+    def names(self):
+        """List git tags in the repository."""
+        try:
+            tags_list = [x.name for x in self.git_repo.repo.tags]
+        except git.GitCommandError as ex:
+            repo_path = self.git_repo.repo.working_dir
+            msg = "Error listing tags for {repo}. Error: {error}".format(error=ex, repo=repo_path)
```

Comment:
Minor, but maybe in next patch, we can switch this to use fstrings?

Reply:
The library is still nominally Python 2-compatible, I think there are a lot of places that need to be updated. It probably should be done all at once, feel free to add notes to https://github.com/release-depot/git_wrapper/issues/32 in relation to this work. And thank you for sanity checking the patch!